routes/runtime/github.rb

```diff
@@ -82,7 +82,7 @@ class CloverRuntime
           fail CloverError.new(409, "AlreadyExists", "A cache entry for #{scope} scope already exists with #{key} key and #{version} version.")
         end
 
-        if size > GithubRepository::CACHE_SIZE_LIMIT
+        if size&.> GithubRepository::CACHE_SIZE_LIMIT
```

Comment:
nit: It's very hard to read

```suggestion
        if size && size > GithubRepository::CACHE_SIZE_LIMIT
```